### PR TITLE
Send/recieve data to/from API

### PR DESCRIPTION
**Pull request by M-AminAlizadeh**
The things that I did in this PR:

1. No linters error
2. Create a new game with the name by using the API.
3. Saved the ID of the game that returned by API in the local storage
4. Create the "Refresh" button that will get all scores for the game from the API (receiving data from the API and parsing the JSON).
5. Create the form "Submit" button that will save a score for the game  (sending data to the API). 
6. Using `Es6 `and `async/await`

### CODE REVIEW

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @M-AminAlizadeh ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [x] You have submitted a project with failing linter checks. Kindly ensure your linter checks are passing and are in green.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — COMMENTED** [review]
# STATUS: INVALID SUBMISSION

Hi @M-AminAlizadeh , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has the wrong review guidelines. You submitted `Send/receive data to/from API` meanwhile the code review guideline is for another milestone: `Leaderboard: Final Touches`. Kindly do the right submission and request for another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -1 +1,25 @@
...
+import resetForm from './Modules/reset.js';
+
+const submitBtn = document.querySelector('.submit-btn');
+const refreshBtn = document.querySelector('.refresh-btn');
+
+if (!gameId) {
+  makeGameId();
+} else {
+  recieveData(gameId);
+  // console.log(recieveData(gameId));
```
Comment (wayungi) [inline src/index.js:14]: - [ ] Do not commit old pieces of code as inline comments. They make your project look messy. If you need to review a previous version of your code, you can always use git history. 😄

File: src/Modules/gameId.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/* eslint-disable */
```
Comment (wayungi) [inline src/Modules/gameId.js:1]: - [ ] Please do not disable linters. Linters ensure that we write uniform code across a team. 🤝

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @M-AminAlizadeh , 

image


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] All linter checks are passing ✔️ 
- [ ] Good use of arrow functions ✔️  
- [ ]  Correct Gitflow ✔️ 
- [ ] Good use of async & await ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (M-AminAlizadeh) [reply src/index.js:14]: Ok

File: src/Modules/gameId.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+let gameId = localStorage.getItem('gameId') || '';// eslint-disable-line
```
Comment (KanzaTahreem) [inline src/Modules/gameId.js:1]: - [ ] As suggested [here](https://github.com/M-AminAlizadeh/Microverse-leaderboard/pull/2#discussion_r1206268264), please do not disable linters. They play a crucial role by improving code quality, maintainability, and collaboration within a development team 🔨🤓

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @M-AminAlizadeh 🙋‍♀️

Good job so far! I really appreciate the effort you put in! **_However, there are few things that still need to be fixed. But you are almost there!_** 💪 🏁

so_close

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good use of `async` & `await`
- ✅ Good branch name
- ✅ Descriptive PR and README file
- ✅ Correct Linters

## Required Changes 🛑

> _Please keep in mind that your long hours of work is appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄_

_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @M-AminAlizadeh 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Linters checks are passing
- ✔️ Use Git Flow
- ✔️ Meaningful commit messages.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**